public/data/common_names.json

```diff
@@ -0,0 +1,2 @@
+{   "ngp": "North Galactic Pole", 
```

Comment:
I was imagining this data inside the function, but I like it as a separate json file even better.